Wikimate.php

```diff
@@ -183,7 +183,8 @@ protected function initRequests()
 	 * @param  array         $headers  Optional extra headers to send with the request
 	 * @param  boolean       $post     True to send a POST request, otherwise GET
 	 * @return Requests_Response       The API response
-	 * @throw  Exception               If lagged and ran out of retries
+	 * @throw  WikimateException       If lagged and ran out of retries,
+	 *                                 or an unexpected API response
```

Comment:
The second clause should include its own verb. For example:

```suggestion
	 *                                 or got an unexpected API response
```

Reply:
Picky picky :smiley_cat: